src/Notifications/AbstractNotification.php

```diff
@@ -63,4 +71,53 @@ abstract class AbstractNotification
 
         return config($config_key, []);
     }
+
+    /**
+     * @var array
+     */
+    protected $tags = [];
+
+    /**
+     * The number of times the job may be attempted.
+     *
+     * @var int
+     */
+    public $tries = 5;
+
+    /**
+     * Assign this job a tag so that Horizon can categorize and allow
+     * for specific tags to be monitored.
+     *
+     * If a job specifies the tags property, that is added.
+     *
+     * @return array
+     */
+    public function tags(): array
+    {
+        $tags = ['seatnotifications'];
+        if (property_exists($this, 'tags'))
```

Comment:
`tags` is always defined since you declare it upper inside the main class.

Reply:
this is based from your SlackJobBase ...

https://github.com/warlof/slackbot/blob/master/src/Jobs/SlackJobBase.php#L55-L63

should i remove it?